mne/decoding/time_gen.py

```diff
@@ -359,7 +354,7 @@ def predict(self, epochs, test_times=None):
                                  for t_test in t_train]
                                  for t_train in test_times_['slices']]
         # Store all testing times parameters
-        self.test_times_ = test_times_
+        self.test_times = test_times_
```

Comment:
this not sklearn consistent. You should have both test_times and test_times_ in order to avoid changing init param during fit.